app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CommentsController < ApplicationController
+  def create
+    @comment = Comment.new(params.require(:comment).permit(:text))
+    @comment.author = current_user
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to "/users/#{current_user.id}/posts/#{params[:post_id]}"
+    else
```

Comment:
- [ ] According to project requirements, the `CommentsController` should have both `new` and `create` actions
- [ ] Following from the above requirement, please create a `views/comments/new.html.erb` file and ensure that it has a form that accepts input for creating comments because we currently can't create comments as you can see below 👇 
![image](https://user-images.githubusercontent.com/46839548/206876870-23958b43-ad50-4b0a-8f46-4803e3f855b8.png)